MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Muon/MuonAnalysis.h

```diff
@@ -257,8 +257,8 @@ private slots:
   void setTFAsymm(Muon::TFAsymmState);
   /// Called when "overwrite" is changed
   void updateDataPresenterOverwrite(int state);
-  // update norm
-  void updateNorm(QString name);
+  // update the displayed normalisation
+  void updateNormalisation(QString name);
```

Comment:
You have mixed spelling of Normalisation, in some functions it's Normali**z**ation, this might confuse IDEs and developers